x/leverage/keeper/msg_server_test.go

```diff
@@ -155,7 +155,7 @@ func (s *IntegrationTestSuite) TestUpdateRegistry() {
 				s.Require().NoError(err)
 				// no tokens should have been deleted
 				tokens := s.app.LeverageKeeper.GetAllRegisteredTokens(s.ctx)
-				s.Require().Len(tokens, 2)
+				s.Require().Len(tokens, 3)
```

Comment:
shall we add a test with decimals=18?

Reply:
No updated test is needed for UpdateRegistry, but several other locations have had tests added or modified to ensure the math is working properly